packages/veritone-react-common/package.json

```diff
@@ -5,23 +5,26 @@
   "module": "dist/bundle-es.js",
   "jsnext:main": "dist/bundle-es.js",
   "dependencies": {
-    "@material-ui/core": "^1.0.0",
-    "@material-ui/icons": "^2.0.0",
+    "@material-ui/core": "^4.3.3",
+    "@material-ui/icons": "^4.2.1",
+    "@material-ui/lab": "^4.0.0-alpha.24",
+    "@material-ui/styles": "^4.3.3",
     "classnames": "^2.2.5",
     "clipboard": "^2.0.4",
     "date-fns": "^1.29.0",
     "google-maps": "^3.3.0",
+    "jss": "10.0.0-alpha.17",
```

Comment:
@SteveShaffer are we ok with alpha version here?

Reply:
I am following this to fix the problem with jss: https://github.com/mui-org/material-ui/issues/12362

Reply:
Yes I'm fine with this case.

btw something to note: our mix of bundled-dependencies-that-all-rely-on-eachother includes React, Redux, Material-UI, and now JSS (and I presume react-jss?)